go/test/endtoend/vtgate/misc_test.go

```diff
@@ -328,11 +328,13 @@ func TestOffsetAndLimitWithOLAP(t *testing.T) {
 	conn, err := mysql.Connect(ctx, &vtParams)
 	require.NoError(t, err)
 	defer conn.Close()
+	defer exec(t, conn, "delete from t1")
```

Comment:
nit: I'd prefer to move the `set workload='oltp'` to the defer so it's clear that they go together